tck/src/main/java/org/eclipse/microprofile/lra/tck/service/LraRecoveryService.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.lra.tck.service;
+
+import org.eclipse.microprofile.lra.tck.spi.LraRecovery;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.net.URI;
+import java.util.Iterator;
+import java.util.ServiceLoader;
+import java.util.logging.Logger;
+
+@ApplicationScoped
+public class LraRecoveryService {
+
+    private static final Logger LOG = Logger.getLogger(LraRecoveryService.class.getName()); 
+    
+    private LraRecovery lrarRecovery = lookupLraRecovery();
```

Comment:
Usage of @PostConstruct since this is a CDI bean?

Reply:
minor: typo lrarRecovery

Reply:
post construct will be run only once the bean is constructed, so I chose to rather use static method which will fail sooner.

Reply:
I wonder if the ServiceLoader is guaranteed to be available during the construction of this class? And assuming it is what guarantees that the service is registered pre the construction of the class - I guess there should be some notes added to the TCK guide about the expectations?

Reply:
@tomjenkinson per discussion on gitter I will replace the service loader in this PR with CDI definition that will be supplied to the Arquillian directly.

Reply:
Good plan, thanks @xstefank